regexTesting.py

```diff
@@ -0,0 +1,137 @@
+from app import register_validation as validation
+from app.utilities.utilities import is_leap_year
+
+import unittest
+
+
+# TODO try pytest later
+class RegexTesting(unittest.TestCase):
+
+    def test_password_validation(self):
+
+        passwords = {
+            'Roszek91' : True,
+            'Tomeczek1991': True,
+            'LubiePlacki12@@@:': True,
+            'Tom91': False,
+            'tomekbomek': False,
+            'TomeczekSromeczek': False}
+
+        msg = "Test {0} expected {1} actual {2}"
+
+        for password, expected in passwords.items():
+            try:
+                actual = validation.validate_password(password)
+                self.assertEqual(actual, expected, msg.format(password, expected, actual))
+            except AssertionError as error:
+                print(error)
+
+    def test_email_validation(self):
```

Comment:
you are using email validation library now, so you can remove your test. You dont test things made by others. You can keep if you want a prove that you worked hard. Just add some comment like: Not really needed as external email validation library is used now

Reply:
done